graaljs/graaljs-gradle-webpack-guide/README.md

```diff
@@ -0,0 +1,187 @@
+# Using Node Packages in a Java Application with Gradle
+
+## Introduction
+This guide demonstrates how to integrate JavaScript libraries into a Java application using **GraalJS** and the **GraalVM Polyglot API**. Specifically, we will use the `qrcode` NPM package to generate QR codes within a Java application, leveraging GraalVM's ability to execute JavaScript code.
```

Comment:
Please add links and format the file so that there is one sentence per line. This simplifies git diffs and avoids some merge conflicts in the future.

```suggestion
This guide demonstrates how to integrate JavaScript libraries into a Java application using [GraalJS](https://www.graalvm.org/javascript) and the [GraalVM Polyglot API](https://www.graalvm.org/latest/reference-manual/embed-languages/).
Specifically, we will use the `qrcode` NPM package to generate QR codes within a Java application, leveraging GraalVM's ability to execute JavaScript code.
```